test/tests/api-cit/v2_0/workflows_tests.py

```diff
@@ -192,90 +211,55 @@ def post_workflows(self, graph_name, timeout_sec=10, nodes=[], data=None, tasks=
-                if graphId == routeId:
-                    if 'target' in w['context']:
-                        nodeid = w['context']['target'] or 'none'
-                    else:
-                        nodeid = 'none'
-                    status = body['status']
-                    if status == 'succeeded' or status == 'failed':
-                        logs.info('%s - target: %s, status: %s, route: %s',
-                                  injectableName, nodeid, status, routeId)
-                        self.__class__.__graph_status.append(status)
-
-                        for task in self.__class__.__tasks:
-                            if task.id == nodeid:
-                                task.worker.stop()
-                                task.running = False
-
-                        msg = {
-                            'graph_name': injectableName,
-                            'target': nodeid,
-                            'status': status,
-                            'route_id': routeId
-                        }
-
-                        if status == 'failed':
-                            msg['active_task'] = w['tasks']
-                            logs.error(dumps(msg, indent=4))
-                        else:
-                            logs.info(dumps(msg, indent=4))
-                        break
-
-    def run_workflow_tasks(self, tasks, timeout_sec):
-        def thread_func(worker, id):
-            worker.start()
-        tasks = self.__class__.__tasks if tasks is None else tasks
-        worker_tasks = WorkerTasks(tasks=self.__class__.__tasks, func=thread_func)
-        worker_tasks.run()
-        worker_tasks.wait_for_completion(timeout_sec=timeout_sec)
-        for task in tasks:
-            if task.timeout:
-                logs.error('Timeout for %s, node %s', self.__class__.__graph_name, task.id)
-                self.__class__.__graph_status.append('failed')
-        if 'failed' in self.__class__.__graph_status:
-            self.fail('Failure running {}'.format(self.__class__.__graph_name))
-
-    @depends(after=['test_delete_all_active_workflows', 'test_workflows_graphs_delete'])
-    def test_node_workflows_post(self):
-        # """Testing POST:/nodes/id/workflows"""
-        self.post_workflows("Graph.noop-example")
+                node_id = n.get('id')
+                self.assertIsNotNone(node_id)
+                logs.debug(' Starting amqp listener for node %s', node_id)
+
+                # make sure to cannel any possible active workflow
+                self.__cancel_active_workflow(node_id)
+
+                # post new workflow
+                Api().nodes_post_workflow_by_id(node_id, name=self.__class__.__graph_name, body=data)
+                logs.info("Posted workflow %s on node %s", self.__class__.__graph_name, node_id)
+                data = self.__get_data()
+                workflow_id = data['instanceId']
+
+                logs.info('******************** wf_id: %s', workflow_id)
```

Comment:
nit: I recall throwing these in with all the '*'s in order to find them in the logs easier. I don't think they should stay way? (and might as well put them both in one line!)